Godot.gitignore

```diff
@@ -0,0 +1,8 @@
+temp/
+.import/
+.mono/
+*.import
```

Comment:
This should not be ignored, it's what defines the import settings for each resource. If you ignore those, your project will always have the default import settings (so say farewell to pixel art).

Reply:
I see some duplicates of `*.import`, should the folder be ignored or the file extension or none? Or just the `*.stex` and `*.oggstr`? I'm bit of confused with these imports extensions.

Also what about the **.mono** folder?

Reply:
- `.import/`, the folder at the root of your project repo (where `project.godot` is), is where Godot stores all the files it imports automatically based on your source assets and their `assetname.import` flags. This should be git-ignored.
- `*.import` files are text files with the flags to tell Godot how to import each asset. They need to be kept, unless you want to reconfigure all the time.